src/components/Select/Select.tsx

```diff
@@ -0,0 +1,41 @@
+type Props = {
+  selectValue: number,
+  setOptionPage: (value: number) => void,
+  resetPage: (page: number) => void,
+};
+
+const selectOptions = [3, 5, 10, 20];
```

Comment:
The array of select options should be a prop to make the component more reusable. If it's defined inside the component, you will be unable to change the options without modifying the component itself.